tests/ctst/common/common.ts

```diff
@@ -66,3 +78,32 @@ Given('an object {string} that {string}',
             ));
         }
     });
+
+Then('kafka consumed messages should not take too much place on disk',
+    async function (this: Zenko) {
+        const notToCheckTopics = ['oplog', 'dead-letter'];
+        const kafkaAdmin = new Kafka({ brokers: [this.parameters.KafkaHosts] }).admin();
+        const topics: string[] = (await kafkaAdmin.listTopics())
+            .filter(t => (t.includes(this.parameters.InstanceID) &&
+            !notToCheckTopics.some(e => t.includes(e))));
+        const previousOffsets = await getTopicsOffsets(topics, kafkaAdmin);
+
+        const seconds = parseInt(this.parameters.KafkaCleanerInterval);
+
+        // Waiting for kafkacleaner to run
+        await Utils.sleep(seconds * 1000);
+
+        const newOffsets = await getTopicsOffsets(topics, kafkaAdmin);
+
+        for (let i = 0; i < topics.length; i++) {
+            process.stdout.write(`\nChecking topic ${topics[i]}\n`);
+            for (let j = 0; j < newOffsets[i].partitions.length; j++) {
+                // Checking that the min offset has increased due to kafkacleaner
+                // or that it didn't need to change because there was no new messages
+                assert.ok(newOffsets[i].partitions[j].low > previousOffsets[i].partitions[j].low ||
+                    newOffsets[i].partitions[j].high === newOffsets[i].partitions[j].low,
+                `Topic ${topics[i]} partition ${j} low offset has not increased, newOffsets:
+                    ${newOffsets[i].partitions[j].low}, previousOffsets: ${previousOffsets[i].partitions[j].low}`);
+            }
```

Comment:
For most topic, the partition should be empty.

There may only be a very few number of topcs which remain active after the tests: bucket-topics for sure (as the conductor will keep polling), but should not mcuh else (if anything).
→ would be better to check strictly, and accept that the topic is empty only if new messages have indeed been pushed in the meantime.